code/controllers/configuration.dm

```diff
@@ -991,3 +995,9 @@ var/list/gamemode_cache = list()
 			config.python_path = "/usr/bin/env python2"
 		else //probably windows, if not this should work anyway
 			config.python_path = "python"
+
+	//Check if the server_id is set
+	if(config.sql_enabled && !server_id)
+		var/msg = "SQL has been enabled, but the server_id has not been set!"
+		admin_notice(span("danger", msg), R_DEBUG)

```

Comment:
There must be a better way to notify the admins when they are actually connected.
Right now that would trigger very early when noone is connected.